From dr.gregory.h.wettstein.phd@ndsu.edu  Wed Apr 15 16:17:36 2009
From: Dr. Greg Wettstein <greg@enjellic.com>
Date: Sat, 11 Apr 2009 09:12:08 -0500
Subject: USB: Add device ID for Motorola v950 'Renegade' phone.
Subject: Re: USB device codes for Motorola phone.
To: Greg KH <greg@kroah.com>
Message-ID: <200904111412.n3BEC8vW007249@habu.cc.ndsu.nodak.edu>

The v950 appears to be a ruggedized version of the Motorola Razor
phone.  Tethering to the phone to use it in 'phone as modem' mode
requires the use of the specialized moto-modem driver which layers
over the usb-serial driver.  Support for the v950 was added simply
adding the device ID's for the phone.

Signed-off-by: Dr. Greg Wettstein <greg@enjellic.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/moto_modem.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/serial/moto_modem.c
+++ b/drivers/usb/serial/moto_modem.c
@@ -25,6 +25,7 @@ static struct usb_device_id id_table [] 
 	{ USB_DEVICE(0x05c6, 0x3197) },	/* unknown Motorola phone */
 	{ USB_DEVICE(0x0c44, 0x0022) },	/* unknown Mororola phone */
 	{ USB_DEVICE(0x22b8, 0x2a64) },	/* Motorola KRZR K1m */
+	{ USB_DEVICE(0x22b8, 0x2c64) }, /* Motorola V950 phone */
 	{ },
 };
 MODULE_DEVICE_TABLE(usb, id_table);
